README.md

```diff
@@ -72,3 +72,27 @@ render((
   </Router>
 ), document.getElementById('root'))
 ```
+
+#### If you're using React Router 4's BrowserRouter you can incorporate qhistory like this:
+
+```js
+class QueryRouter extends React.Component {
+  static propTypes = {
+    basename: PropTypes.string,
+    forceRefresh: PropTypes.bool,
+    getUserConfirmation: PropTypes.func,
+    keyLength: PropTypes.number,
+    children: PropTypes.node,
+  }
+
+  history = qhistory(
+    createBrowserHistory(this.props),
+    stringifyQueryParams,
+    parseQueryParams
```

Comment:
I'll merge this if I can get you to make two more changes:

1. Make the `stringify` and `parse` functions passed to `qhistory` props.
2. Add a usage example at the bottom of this snippet, e.g.
```js
// usage
render((
  <QueryRouter stringify={stringify} parse={parse}>
    <App />
  </QueryRouter>
), document.getElementbyId('root'))
```